RSSReader/src/app/RSS_GUI.fxml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import java.util.*?>
+<?import javafx.scene.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+
+<AnchorPane id="AnchorPane" prefHeight="537.0" prefWidth="947.0" xmlns:fx="http://javafx.com/fxml/1" xmlns="http://javafx.com/javafx/8">
+   <children>
+    <AnchorPane layoutX="278.0" layoutY="253.0" minHeight="0.0" minWidth="0.0" prefHeight="200.0" prefWidth="200.0" />
+      <ScrollPane layoutX="30.0" layoutY="62.0" prefHeight="452.0" prefWidth="137.0">
+         <content>
+            <TextArea prefHeight="452.0" prefWidth="124.0" />
```

Comment:
Should not be editable, but not a big deal for what we're doing here. There's a checkbox in SceneBuilder.